setup_logging.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+function setup_logging() {
+    echo "installing Cluster Logging and EFK operators..."
+    oc create -f cluster-logging/eo-namespace.yaml
+    oc create -f cluster-logging/clo-namespace.yaml
+    oc create -f cluster-logging/eo-og.yaml
+    oc create -f cluster-logging/eo-sub.yaml
+    oc create -f cluster-logging/clo-og.yaml
+    oc create -f cluster-logging/clo-sub.yaml
+    # wait until the CRD is available before creating the CR
+    NEXT_WAIT_TIME=0
+    while [[ -z `oc get crd clusterloggings.logging.openshift.io 2>/dev/null` ]]; do
+		if [[ ${NEXT_WAIT_TIME} -eq 300 ]]; then \
```

Comment:
```suggestion
		if [[ ${NEXT_WAIT_TIME} -eq 300 ]]; then
```
the slash is not needed here